sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+# CNCF Special Interest Groups ("SIGs")
+
+Proposal by the CNCF TOC and Contributors
+Primary Authors: Alexis Richardson, Quinton Hoole
+November 2018 - January 2019
+
+Final Draft v1.0
+
+[[TOC]]
+
+## Overall Purpose
+
+Scale contributions by the CNCF technical and user community, while retaining integrity and increasing quality in support of our [mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation). 
+
+## Specific Objectives
```

Comment:
nit: some have periods, some do not - let's be consistent

Reply:
Done. Added 3 missing periods.